.github/workflows/release.yml

```diff
@@ -9,7 +9,6 @@ on:
       - "v[0-9]+.[0-9]+.[0-9]+-[0-9]+" # ex. v0.29.9-1
       - "v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+" # ex. v0.29.9-1-rc1
 env:
-  RUST_TOOLCHAIN: nightly-2022-07-23
   RELEASE_VERSION: ${{github.ref_name}}
```

Comment:
Remove ust toolchains from workflows as we specify the rust-toolchain.toml which keeps local and build env same